Rohit/PostgresAssignment/controller/result.ts

```diff
@@ -0,0 +1,131 @@
+import { Context } from "vm";
+
+const res = require('../sql/result');
+
+interface resultinfo {
+    result_id: number;
+    studentid: number;
+    clas_id: number;
+    subjectid: number;
+    marks: number;
+  }
+
+  
+exports.resultData = async (ctx: Context) => {
+    try {
+      let [rows]: Array<{ rows: resultinfo }> = [];
+      rows = await res.get_result();
```

Comment:
are we sending all results here, we need result for student, highst marks for given subject/class. we dont need all results